ecommerce/user/utils.py

```diff
@@ -0,0 +1,113 @@
+from app.database import get_db
+from data.model import Data
+from user.model import Users
+from auth.schemas import UserOut
+from user.password import hash_password
+from sqlmodel import select
+from auth.json_token import SECRET_KEY, ALGORITHM
+
+from jose import jwt, JWTError
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+
+
+router = APIRouter()
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl='login',
+                                     scheme_name="JWT")

```

Comment:
```suggestion
oauth2_scheme = OAuth2PasswordBearer(
    tokenUrl='login',
    scheme_name="JWT",
)
```
Bad formatting, I would recommend using `isort` and `black` code formatters